test/e2e/lib/components/sidebar-component.js

```diff
@@ -55,6 +55,13 @@ export default class SidebarComponent extends AsyncBaseContainer {
 		); // TODO: data-tip-target target is missing
 	}
 
+	async selectWPAdmin() {
+		return await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css( '.menu-link-text[data-e2e-sidebar="WP Admin"]' )
+		); // TODO: data-tip-target target is missing
```

Comment:
Looking through the code for this, it seems like `this._scrollToAndClickMenuItem()` that would use the `data-tip-target` is a lot more robust. We have been using `selectThemes` in the inline help e2e tests and I've noticed it being flakey.

Reply:
Ran this a couple times and it would fail around this sidebar selection sometimes.

Reply:
Great spot. This is super flaky whereas the scroll to method is much more robust. I've updated the code to include the data-tip-target necessary to utilise this method and removed the previous change.